packages/editor/src/components/post-author/enhanced-post-author.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import { debounce } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import ComboboxControl from '../../../../components/build/combobox-control/';
+
+function EnhancedPostAuthor( { postAuthor, id, authors, onUpdateAuthor } ) {
+	const currentPostAuthor = postAuthor[ 0 ];
+	let initialAuthors = [];
+	authors.forEach( ( author, i ) => {
+		initialAuthors = [
+			...initialAuthors,
+			{
+				key: i,
+				name: author.name,
+				value: author.name,
+				id: author.id,
+			},
+		];
+	} );
```

Comment:
This should be memoized and use `.map`, not `.forEach`.

Reply:
Ok - why may vs. forEach? I had map originally but we aren't returning anything so why the preference for map?

Reply:
switched to map, that is cleaner. Somehow didn't work originally.